newIDE/app/src/ProjectCreation/CreateProjectDialog.js

```diff
@@ -92,56 +76,22 @@ const CreateProjectDialog = ({
 
   const actions = React.useMemo(
     () => [
-      <DialogPrimaryButton
-        key="create-blank"
-        label={<Trans>Create a blank project</Trans>}
-        primary={false}
-        onClick={() => setPreCreationDialogOpen(true)}
-      />,
       <FlatButton
         key="close"
         label={<Trans>Close</Trans>}
         primary={false}
         onClick={onClose}
       />,
+      <DialogPrimaryButton
+        key="create-blank"
+        label={<Trans>Create a blank project</Trans>}
+        primary
+        onClick={() => setPreCreationDialogOpen(true)}
+      />,
     ],
     [onClose, setPreCreationDialogOpen]
   );
 
-  const secondaryActions = React.useMemo(
-    () => {
-      if (currentTab === 'games-showcase')
-        return [
-          <FlatButton
-            key="submit-game-showcase"
-            onClick={() => {
-              Window.openExternalURL(
-                'https://docs.google.com/forms/d/e/1FAIpQLSfjiOnkbODuPifSGuzxYY61vB5kyMWdTZSSqkJsv3H6ePRTQA/viewform?usp=sf_link'
-              );
-            }}
-            primary
-            icon={<PublishIcon />}
-            label={<Trans>Submit your game to the showcase</Trans>}
```

Comment:
Same as for game showcase submission

Reply:
NB: both are hidden on small screens as:
- it's hard to make them fit on small screens!
- I don't expect users to submit game/example when they're on their mobile device